UIKit/Tasks/StackViewController.swift

```diff
@@ -0,0 +1,58 @@
+// StackViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран создание сфетофора с помощью StckView
+final class StackViewController: UIViewController {
+    // MARK: - Visual Components
+
+    private let redView = CustomView()
+    private let yellowView = CustomView()
+    private let greenView = CustomView()
+    private let grayView = UIView()
+    private var stackView = UIStackView()
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+        setupStackViewConstraints()
+        setupUIGrayViewConstraints()
+    }
+
+    // MARK: - Private Methods
+
+    private func setupUI() {
```

Comment:
```suggestion
    private func configureUI() {
```
Так просто договорить писать вроде как